about.html

```diff
@@ -0,0 +1,115 @@
+        <div class="footer">
+        <p class="date">Please contact us per Email for nay further questions about Festival! <p>
+           <p class="address"> Kamrankhan.ahmad2@yahoo.com</p>
+        </div>
+    </div>  
+    </main>
+    </header>
+    
+    <div class="background-img3">
+        <div class="main-logo">    
+            <h1 class="logo-header">Sports Festival 2022 logo</h1>
+            <img class="bottom-line2" src="images/Underline.png" alt="underline image">
+            </div>
+                <div class="logo-Description">
+                    <p class="description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Repellat necessitatibus autem quidem maiores quaerat ab corrupti error. Deserunt necessitatibus reprehenderit doloribus! Assumenda debitis nulla illo deserunt ad cum dolorem? Suscipit. Lorem ipsum dolor sit amet consectetur adipisicing elit.  </p>
+                    <img class="Des-logo" src="images/logo.png">
+                </div>               
+            </div>
+              
+            <div class="background-img3" id="background-img10">
+                <div class="main-logo" id="main-logo">    
+                    <h1 class="logo-header" id="logo-header">See the past Gala sports Festival</h1>
+                    <img class="bottom-line5" id="line2" src="images/Underline.png" alt="underline image">
+                    </div>
+                        <div class="logo-Description">
+                            <p class="description description2">Take a look at the past two Gala sports Festival which took place in Nangarhar and in Herat </p>
+                            <div class="pair"> 
+                                                       
+                            <img class="Des-logo" src="images/cricket-img.jpg">
+                            <img class="Des-logo" src="images/Football-img.jpg">
+                        </div>
+ 
+                        </div>               
+                    </div>
+           
+          
+   <div class="background-img4">
+    <h2 class="background-img4th"></h2>
+     
+        <div class="partners">    
+        <h1 class="partner">Partners</h1>
+        <img class="bottom-line3" src="images/Underline.png" alt="underline image">
+        </div>
+        <div class="partner-logos">       
+        <img class="partner-logo" src="images/AGC logo.jpg" alt="Financial-supporters"> 
+        <img class="partner-logo" src="images/AWCC.jpg" alt="Financial-supporters">  
+        <img class="partner-logo" src="images/ATN.jpg" alt="Financial-supporter"> 
+        <img class="partner-logo" src="images/IoGlobal.png" alt="Financial-supporter">  
+        </div>
+  
+</div>
+ 
+
+    <div class="background-img5">  
+          <img class="logo2" src="images/logo.png" alt="underline image">
+            <p class="rights">2022 Sports Galla, Kabul -All rights reserved</p>
+            </div>
+
+  
```

Comment:
- [ ] Please wrap your footer section in the `<footer>` semantic tag 👍 This helps for accessibility and SEO